ChangeLog.md

```diff
@@ -1,3 +1,15 @@
+## 5.4.0 (2022-09-20)
```

Comment:
We will need to update, but this is a good placeholder.

Reply:
I moved all the 5.4 references to a separate PR